styles/styles.css

```diff
@@ -14,7 +14,139 @@ body {
 }
 
 .container {
-  padding: 120px 104px 120px 104px;
+  padding: 0 104px 0 104px;
+  margin: 0 auto;
+}
+
+.block-padding {
```

Comment:
Not good name, and here you rewrite padding properties from class `.container`. So better remove it from here and add it to class `.container`